tools/src/commands/GenerateBareApp.ts

```diff
@@ -0,0 +1,133 @@
+import path from 'path';
+import fs from 'fs-extra';
+import { Command } from '@expo/commander';
+import spawnAsync from '@expo/spawn-async';
+import { runExpoCliAsync } from '../ExpoCLI';
```

Comment:
Doesn't linter require an empty line between dependencies and local imports? 🤔 The dependencies also doesn't seem to be sorted.

```suggestion

import { runExpoCliAsync } from '../ExpoCLI';
```